notebooks/energy_volume_curve.ipynb

```diff
@@ -30,14 +30,28 @@
     "ExecuteTime": {
```

Comment:
<p>Gpaw is not a mandatory requirement for the installation, so it would be good to remind readers how to install it, either by pointing to the install link or explicitly mentioning <code>conda install -c conda-forge gpaw</code></p><p> </p><p>Nit: Since "module" has a technical meaning in python, I would prefer to use "the Murnaghan job can be used"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyiron/pyiron/pull/1479/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I might prefer it to be shown in the error message, if Gpaw is used without being installed. But I don't have a very strong feeling, so I can also add it in the notebook.

Reply:
That's a good point. If it's already in the error message then no further action is required, otherwise touching up the gpaw failure message in this PR would be good 👍